docs/reference/package-control/client-usage.md

```diff
@@ -0,0 +1,122 @@
+---
+title: "Package Control Client: Usage"
+---
+
+<!-- Originals: -->
+<!-- https://packagecontrol.io/docs/usage -->
+<!-- https://github.com/wbond/packagecontrol.io/blob/master/app/html/docs/usage.html -->
+
+
+# Client package: Usage
+
+Package Control is driven by the [Command Palette][cmnd]. All Package Control commands begin with "Package Control:", so start by typing "Package".
+
+[cmnd]: command_palette.html#how-to-use-the-command-palette
+
+The command palette will show a number of commands.
+
+## Most-used commands
+
+### Install Package
+
+Show a list of all packages that are available for installation.
+This will include all of the packages from the [default channel][channel],
+plus any from [repositories][repos] you have added.
+
+[channel]: https://github.com/wbond/package_control_channel
+[repos]: repository.html
+
+
+### Add Repository
```

Comment:
[Naming things is hard](https://martinfowler.com/bliki/TwoHardThings.html) ... Anyway, this bit has always confused me. Here "repository" actually means _both_ things, both git repos, and repository jsons are supported. I hope I described it well, the [existing docs](https://packagecontrol.io/docs/usage) are no help.

Reply:
That's probably the way to go. I just went through the docs again to make sure in each case it's clearly either the JSON or the Git repository we're talking about.